fixtures/american-dj/uv-eco-bar.json

```
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "UV Eco Bar",
+  "shortName": "uv-eco-bar",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Nikita"],
+    "createDate": "2024-09-03",
+    "lastModifyDate": "2024-09-03"
+  },
+  "comment": "18x3W UV LEDs",
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/6914/ADJ%20Eco%20UV%20Bar%20DMX%20-%20User%20Manual.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/eco-uv-bar-dmx"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=mEH-cVF_iB4"
+    ]
+  },
+  "physical": {
+    "dimensions": [1000, 60, 115],
+    "weight": 2.4,
+    "power": 60,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 10000
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%",
+        "comment": "UV light intensity"
```

```suggestion
        "type": "ColorIntensity",
        "color": "UV"
```

It's not clear what OFL wants for a non-color-changing but non-white fixture, which doesn't have a separate dedicated Dimmer channel.  For `eurolite/led-sls-6-uv-floor.json`, a ColorIntensity channel was used, so I'd do the same here.